src/app/models/home-buttton-model.ts

```diff
@@ -1,10 +1,8 @@
 export interface homeButtons {
   firstRow: rows[],
   secondRow: rows[]
-  }
-
-  interface rows {
-    title: string,
-    imgPath: string
-  }
-  
```

Comment:
Please bring the new line back.
It was being used to split the interfaces code.

Reply:
I already did that in the second commit.